npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -316,6 +341,10 @@ func (iMgr *IPSetManager) AddToLists(listMetadatas, setMetadatas []*IPSetMetadat
 		// 3. add all members to the list
 		for _, memberMetadata := range setMetadatas {
 			memberName := memberMetadata.GetPrefixName()
+			if memberName == "" {
+				metrics.SendErrorLogAndMetric(util.IpsmID, "[AddToLists] warning: adding invalid member to list %s, member %s", list.Name, memberName)
+				continue
+			}
 			// the member shouldn't be the list itself, but this is satisfied since we already asserted that the member is a HashSet
 			if list.hasMember(memberName) {
```

Comment:
like removeFromList, we could skip member validation and throwing an error above because we'll continue here if it's invalid

Reply:
Same as above.